.github/workflows/ci.yml

```
@@ -2188,10 +2257,25 @@ jobs:
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
         run: pnpm --filter=@jovie/web exec playwright install chromium --with-deps
 
-      - name: Build app for smoke tests
+      - name: Download build artifact from ci-build
+        id: download-build
+        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' && needs.ci-build.outputs.run_full_ci == 'true' }}
+        uses: actions/download-artifact@70fc10c6e5e1ce46ad2ea6f2b72d43f7d47b13c3 # v8.0.0
```

YAMLlint (1.38.0)

[warning] too few spaces before comment: expected 2

(comments)